src/app/loading.tsx

```diff
@@ -1,7 +1,11 @@
 import React from "react";
 
 const Loading: React.FC<{}> = () => {
-    return <h1> Loading </h1>;
+    return (
+        <main>
+            <h1> Loading </h1>
+        </main>
+    );
```

Comment:
That's fair - if this is being used at a component-level, then there should be a separate `loading.tsx` in the component (if this works) that doesn't have the `main` tag on it

Reply:
`main` tag has been removed

Reply:
should it be? I think it requires a `main` tag, because it acts for page-level loading (i.e. mains will be included in the async rendered HTML). Maybe we should move the `main` tag somewhere else? (Also, for myself, loading page maybe should have an accessibility check!)